js-react/README.md

```diff
@@ -1,12 +1,17 @@
-# Run your C++ code on the web: Part X - Interactive form with React
+# C++ on the web: Interactive form with React
 
 In the [previous blog post](TODO add link) we compiled the C++ algorithm into a webassembly code. In this blog post we will create a web application using [React](https://reactjs.org/). The web application will have a web-form which allows us to change the parameters of the algorithm.
 
+I can feel your pain: there are too many things to learn, too many skills to get but too little time available which reminds me our extra-ordinary friend [Napoleon Dynamite](https://www.imdb.com/title/tt0374900/). This blog post will guide you through the process of making a React web application whithout getting lost.
+
+![dynamite_gosh.png](dynamite_gosh.png)
```

Comment:
Where did this original image come from?

Reply:
I made it. :)

Reply:
But you used a background image. I think we need to point to the source of the background image?

Reply:
Lets fix this in #81